packages/database/src/migrations/20210119053017-SomoaCovidCustomisedRawDataDownload-modifies-data.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+import { insertObject, addReportToGroups } from '../utilities/migration';
```

Comment:
About `addReportToGroups` : We shouldn't import DB specific utilities from `migration.js`, but rather copy them in the migration - see [this comment](https://github.com/beyondessential/tupaia/blob/dev/packages/database/src/utilities/migration.js#L141)